plugins/inputs/chrony/chrony_test.go

```diff
@@ -40,6 +40,10 @@ func TestGather(t *testing.T) {
 		"update_interval": 507.2,
 	}
 
+	// tests on linux with go1.20 will add a warning about code coverage
+	// due to the code coverage dir not being set
+	delete(acc.Metrics[0].Tags, "warning")
+
```

Comment:
Sorry, but how does this comment relate to the code?

Reply:
If you run the `TestGather` test above this section [in CircleCI](https://app.circleci.com/pipelines/github/influxdata/telegraf/14791/workflows/65cdd38e-1c74-44b9-b480-4ea34b99331d/jobs/233808), there will be an additional tag added called "warning" and with the value "[could not set code coverage](warning:gocoverdir not set, no coverage data emitted)".

The additional coverage output is caused when running the `fakeExecCommand` function below, and this plugin will parse the `key: value` expression adding it as a tag.

Reply:
Understood. Fine with me then...